__e2e__/lib/constants.js

```diff
@@ -1,6 +1,6 @@
 // TODO: Pull these from app constants, and add '.' programmatically.
 export const WAIT_FOR_EXIST_TIMEOUT = 20000;
-export const DEFAULT_TIMEOUT_INTERVAL = 35000;
+export const DEFAULT_TIMEOUT_INTERVAL = 55000;
```

Comment:
@hunterlester I'd tweak this value as it's pulled into each test instead of changing the overall default timeout